Misc/NEWS.d/next/Core and Builtins/2019-07-22-11-05-05.bpo-37648.6TY2L-.rst

```diff
@@ -0,0 +1,3 @@
+Fixed minor inconsistency in ``list.__contains__()``, ``tuple.__contains__``
```

Comment:
Fixed trivial inconsistency with ``tuple.__contains__()`` missing parens and added inline markup for the class methods. Since this will cause the line width to go from 76 to 86, the last word will likely have to be shifted down to the next line.

```suggestion
Fixed minor inconsistency in :meth:`list.__contains__()`, :meth:`tuple.__contains__()`
```